src/development/ui/advanced/typography.md

```diff
@@ -0,0 +1,32 @@
+---
+title: Flutter's fonts and typography
+description: Learn about Flutter's support for typography.
+---
+
+If you aren't familiar with [_typography_][], it essentially
+means how text is styled. [Variable fonts][] (also
+called OpenType fonts), allow you to control various
+aspects of text styling, such as font weight, width, slant,
+and so on.
+
+## API docs
+
+Here are some Flutter classes that allow you to
+modify variable fonts:
```

Comment:
**issue:** Avoid use of here to indicate position in the document. 
```suggestion
The following Flutter classes that allow you to
modify variable fonts:
```

Reply:
Yeah, both of these wordings are removed.